core/src/test/scala/org/apache/spark/deploy/StandaloneDynamicAllocationSuite.scala

```diff
@@ -455,6 +482,19 @@ class StandaloneDynamicAllocationSuite
     sc.killExecutors(getExecutorIds(sc).take(n))
   }
 
+  private def killExecutorWithForce(
+      sc: SparkContext,
+      executorId: String,
+      force: Boolean = true): Boolean = {
```

Comment:
I did that (i.e., syncExecutors) outside of this function. since this function is only used in newly added test, that is why not to choose the name of `killExecutor`.